docs/user-guide/global-configurations/catalog-framework.md

```diff
@@ -0,0 +1,85 @@
+# Catalog Framework
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+Ideally, a microservice/application should contain detailed information so that its users know what it does, how to use it, as well as all its technical specs. Access to such data makes it easier for engineers to quickly discover relevant microservices and also its dependent services. 
+
+With Catalog Framework, you as a [super-admin](../global-configurations/authorization/user-access.md#role-based-access-levels) can decide the data you expect from the managers of different resource types. In other words, you can create a whole JSON schema that would ultimately render a form for the resource owners to fill.
+
+Therefore, there are two parts involved in the creation of a desirable resource catalog:
+
+* [Defining a Schema](#defining-a-schema) 
+* [Filling the Details](#filling-the-details)
```

Comment:
"Filling the required details in a form generated based on the defined schema". I think something like this helps connect the two steps better. Would be great if we could shorten this while retaining the essence.